docs/Change_Log.md

```diff
@@ -1,10 +1,35 @@
 # Change Log
 
+## Changes to Bot Permissions for Interactions and Webhooks
+
+> danger
+> This entry includes breaking changes
+
+#### Jun 29, 2022
+
+#### Upcoming Changes
+
+Starting **July 28, 2022**, the way a bot's [permissions](#DOCS_TOPICS_PERMISSIONS/permissions) are calculated is changing in two cases:
+- When **responding to an [interaction](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING)** (like application commands or message components)
+- When **executing a [webhook](#DOCS_RESOURCES_WEBHOOK) that the bot created**
+
+Going forward, in the above cases, a bot’s permissions will be calculated using the permissions granted to it, including any [overwrites](#DOCS_TOPICS_PERMISSIONS/permission-overwrites). This change makes a bot's permissions for interactions and webhooks the same as those for [posting a message to a channel](#DOCS_RESOURCES_CHANNEL/create-message). Previously, a bot’s permissions in these cases relied only on those granted to `@everyone`.
+
+At the time of the change, **Attach Files** (`ATTACH_FILES`), **Embed Links** (`EMBED_LINKS`), **Mention Everyone** (`MENTION_EVERYONE`), and **Use External Emojis** (`USE_EXTERNAL_EMOJIS`) are the permissions that may be affected.
+
+This change *only* applies to bots. The permissions for an app without a bot user (or without the `bot` scope) will still depend on `@everyone`.
+
+#### Updating Your App
+
+If your bot wants to use any affected permissions (`ATTACH_FILES`, `EMBED_LINKS`, `MENTION_EVERYONE`, or `USE_EXTERNAL_EMOJIS`) when responding to interactions or executing a webhook, **ensure that the bot was installed with them**. If it wasn’t, your app with the bot should be reauthenticated with them before July 28.
```

Comment:
This implies that the @ everyone calculated permissions will not be considered even if the bot isn't in the guild.

Reply:
I don't really agree that it implies that, especially with the context provided in the above section. Unless a dev relies on the `@everyone` permissions grant their bot`ATTACH_FILES`, `EMBED_LINKS`, `MENTION_EVERYONE`, or/and `USE_EXTERNAL_EMOJIS` in every server its installed, then a dev should make sure their bot is installed with the permissions.

The section above is the technical changes, the **Updating your app** section is purely to give devs advice on the practical changes they should consider to mitigate breaking changes.